pkgs/servers/web-apps/engelsystem/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  stdenv
+, fetchzip
+, php
+}:
+
+stdenv.mkDerivation rec {
+  pname = "engelsystem";
+  version = "3.1.0";
+
+  src = fetchzip {
+    url = "https://github.com/engelsystem/engelsystem/releases/download/v3.1.0/engelsystem-v3.1.0.zip";
+    sha256 = "01wra7li7n5kn1l6xkrmw4vlvvyqh089zs43qzn98hj0mw8gw7ai";
+    extraPostFetch = "chmod -R a-w $out";
```

Comment:
Normally this sort of thing is done in postUnpack.

Reply:
that refuses to evaluate becaus of suspicious ownership or permission

Reply:
So do we want to keep it in extraPostFetch, or move it to somewhere else?

Reply:
Maybe add a comment about why it's necessary, but I don't see a way around using extraPostFetch.

Reply:
added a comment, will see it as resolved